config.js

```diff
@@ -6,6 +6,7 @@ const DEFAULT_RESPONSE_TIME_QUERY = 'SELECT pg_sleep(1)';
 const DEFAULT_PROGRESS_SCHEDULE = '0 */10 * * * *';
 const DEFAULT_CACHE_HIT_RATIO_SCHEDULE = '0 */10 * * * *';
 const DEFAULT_QUERIES_METRIC_SCHEDULE = '*/10 * * * * *';
+const DEFAULT_BLOCKING_QUERIES_SCHEDULE = '*/10 * * * * *';
```

Comment:
Pour être moins verbeux on peut utiliser par defaut un délai de 10 minutes, à la seconde 0 de chacune de ces dizaines de minutes
```suggestion
const DEFAULT_BLOCKING_QUERIES_SCHEDULE = '0 */10 * * * *';
```